### PR TITLE
PSIRT/VA fix for 5.0 - Go 1.19.6, UBI - 8.7-1085

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -3,6 +3,15 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "7",
+                "release-date": "07 March 2023",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Set release date as 6 March

Comment (GunaKKIBM) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Done

**Review by ashimagarg27 — APPROVED** [review]

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**